pkg/asset-manager-utils/test/AssetManager/AssetManager.test.ts

```diff
@@ -0,0 +1,259 @@
+import { ethers } from 'hardhat';
```

Comment:
What's the rationale for moving this file?  Seems the test folder should have the same, or flatter, structure than the contracts folder.

tbh just mentioning cause git isn't recognizing as a file move so it's hard to tell what's changed